packages/v4/src/content/design-guidelines/components/form/forms.md

```diff
@@ -22,273 +22,375 @@ related: [
-Forms in a page should be left and top aligned in the content area of a page. There should be a minimum of 24px padding around the contents of the form and the neighboring elements.
+### Forms on a page
 
-<img src="./img/page-form.png" alt="Example of form in a page" />
+Left and top align a form within a page’s content area. A minimum of 24px padding should separate the form from neighboring elements.
 
-#### Width
-Generally, it is up to you to decide how wide you want your form to be, but here are some guidelines and recommendations:
-* The width of your form should generally reflect the length of the content that's being inputted.
-* Your form width should be as wide as the widest input field in the form. 
-* If your input fields are all very long, the maximum suggested width is 880 pixels. However, you should use your best judgement based on the context and content of the form. Having a form that is 880px wide is usually not necessary.
+<img src="./img/page-form.png" alt="Example of a form on a page"/>
 
-Note that all input fields in your form will be the same length. The only exception being in a [multi-column form](#multi-column-forms), where you may have two or more adjacent fields in a single line. In that case, the form inputs will share the available width on that line.
+When placing a form on a page, its width is customizable to your use case and should reflect factors including:
 
-### Forms in wizard
-Similar to an in-page form, forms in wizards should be aligned to the left of the content area and have a minimum of 24px padding around its contents and the neighboring elements. This would look the same whether you are using an in-page wizard or a modal wizard.
+* The length of inputted content.
+* The widest input field in the form.
 
-Additionally, error validation for a form in a wizard’s page should occur when or by the time the user clicks the “Next” button. You should not wait until the user has completed the whole wizard before telling them where possible errors lie.
+To accommodate lengthy input fields, the maximum suggested width is 880px. Use your best judgement based on the context and content of your form –– 880px is typically too wide for most use cases.
 
-<img src="./img/wizard-form.png" alt="Example of form in an in-page wizard" />
+All input fields in your form will be the same length, except for in a [multi-column form](#multi-column-forms) where you may place two or more adjacent fields in a single line. In a multi-column form, form inputs will share available width on that line.
+
+### Forms in a wizard
+Similar to an in-page form, forms in wizards should be left-aligned in the content area and have a minimum of 24px padding separating its content from neighboring elements. 
+
+In a wizard, form error validation for a form in a wizard should occur when or by the time the user clicks the “Next” button. Don’t wait until the user has completed the whole wizard to notify them of possible errors and their location.
```

Comment:
Could cut out the first "in a wizard" on second "in a wizard" because it feels repetitive. 
So, "In a wizard, form error validation should occur when...."